src/Park.js

```diff
@@ -26,14 +27,16 @@ class Park extends Component {
     return (
       <div className="card-overlay">
         <div className="park-card-small"> 
-          <p onClick={this.removeCard}>X</p>
+          <i className="far fa-times-circle" onClick={this.removeCard}></i>
           <h1 className="park-title">{this.props.selectedPark.parkName} National Park</h1>
           <img className="park-img" src={imagePath} />
           {/* {imageSource} */}
           <h3 className="park-text">{this.props.selectedPark.description.substring(0, 80)}...</h3>
           <button className="button-small" onClick={this.showFullCard}>View More</button>
-          <p>Visited</p>
-          <p>Bucket</p>
+          <div className="user-list-btns">
+          <Buttons iconType="fas fa-hiking" toolTipText="Add to Visited Parks" toolTipSide="left" parkUrl={this.props.selectedPark.urlCode} storageKey="visitedParks"/>
```

Comment:
Yay! the font awesome icons are working on my computer for once! Thanks for setting these up.